README.md

```diff
@@ -46,52 +46,69 @@ database prior to installing sciSQL. If you do not, YOU WILL LOSE DATA.
 
 If you wish to build/install only the sciSQL client utilities and documentation,
-run configure with the `--client-only` option. In this case, a MySQL server or
+run configure with the `--client-only` option. In this case, a MySQL/MariaDB server or
 client install is not required, the only executable generated is `scisql_index`
 (a utility which generates HTM indexes for tables of circles or polygons stored
-in tab-separated-value format), and `--prefix` can be set to a directory of your
-choice.
+in tab-separated-value format).
 
-Building and installation
--------------------------
+Building
+--------
 
-sciSQL is built, installed, and uninstalled with the usual make,
-make install, and make uninstall commands.
-
-The install command will CREATE the sciSQL UDFs, stored procedures, and
-databases (including the `scisql_demo` database). It will also automatically
-run the sciSQL unit tests. You can re-run the tests anytime with `make test`.
+sciSQL is built and staged with the usual `make` and `make install` commands.
 
 You may wish to regenerate the HTML documentation if you've chosen a
 non-default value for `--scisql-prefix`, as the HTML distributed with release
 tar-balls is built under the assumption that `--scisql-prefix="scisql_"`.
 To do this, run `make html_docs`.
 
+Deploying sciSQL in a MySQL/MariaDB instance
+--------------------------------------------
+
+Assuming you've installed scisql in `$PREFIX`, update your `PATH` and
+`PYTHONPATH` as described below:
+
+    export PATH="$PREFIX/bin:$PATH"
+    export PYTHONPATH="$PREFIX/python:$PATH"
+
+Check that your have access to a local server instance and run `scisql-deploy.py`. Passing
+`--help` will yield a list of configuration options. Here are the ones most likely to require
+tweaking:
+
+| Option               | Description                                                              |
+| -------------------- | ------------------------------------------------------------------------ |
+| `--mysql-user`       | Set this to the name of a MySQL/MariaDB admin user; defaults to `root`.  |
+| `--mysql-bin`        | Point to the `mysql` or `mariadb` command-line client                    |
+| `--mysql-socket`     | Point to the MySQL/MariaDB server UNIX socket file        |
+| `--mysql-plugin-dir` | Point to server plugin directory, typically `/usr/lib/mysql/plugin`      |
+| `--verbose`          | Verbosity level; possible value are FATAL, ERROR, WARN, INFO, DEBUG      |
+
+You will be prompted for the MySQL/MariaDB admin user password during configuration.
+If you run `scisql-deploy.py` in a script, you can
+use standard input, for example via a pipe, for providing this password.
+Connection details, including this password, are stored in a temporary directory in a file named
+`my-client.cnf` using the MySQL options file format. This temporary file is removed at the end
+of the process, unless you set the verbosity level to DEBUG
```

Comment:
Missing period at the end of the sentence.

Reply:
will fix, thx!